src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests_AwaitCompletion.vb

```diff
@@ -320,7 +320,8 @@ public class C
         End Function
 
         <WpfFact>
-        Public Async Function AwaitCompletionDoesNotAddAsync_NotTask() As Task
+        <WorkItem(55975, "https://github.com/dotnet/roslyn/issues/55975")>
+        Public Async Function AwaitCompletionDoesNotAddAsync_Void() As Task
             Using state = TestStateFactory.CreateCSharpTestState(
```

Comment:
Given @CyrusNajmabadi comment that async should be added. We need to rename the test. I think the initial implementation wasn't adding async but later it changed and I forgot to rename the test :)